### DISCUSSION TITLE
Pig Pack

### DISCUSSION THREAD

**Discussion by Jab125** [RedRain0o0/legacy-skins#9] [(C) Meme and Small Skin Packs]
Made in 10 minutes:
![image](https://github.com/user-attachments/assets/a5459a9c-430f-46d2-9603-7d497d25b7a2)
[Pig.zip](https://github.com/user-attachments/files/16893820/Pig.zip)

**Comment by CursedBM81** [comment]
Muddy Pig Update when?

**Comment by jakeyhanna** [comment]
How did you make the model appear in the menu??

When i try to make a pack with my cpmmodel the skin doesn't render but it works fine
